weapons.xml

```diff
@@ -0,0 +1,2164 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Items>
+  <CraftedItem id="crpg_peasant_maul_t1" name="{=nTc20R1R}Makeshift Sledgehammer" crafting_template="crpg_TwoHandedMace" is_merchandise="false" culture="Culture.battania" modifier_group="cheap_weapon">

```

Comment:
you created a new weapons.xml in the wrong folders

Reply:
Should be fixed now